src/App.tsx

```diff
@@ -1,15 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const lowerText = query.toLowerCase();
+
+  const filteredMovies = moviesFromServer.filter(({ title, description }) => (
+    title.toLowerCase().includes(lowerText)
+    || description.toLowerCase().includes(lowerText)
```

Comment:
if some code written twice it can be placed in one function.
Create function for your `.toLowerCase().includes(query.toLowerCase())`